packages/website/ts/utils/constants.ts

```diff
@@ -104,8 +104,9 @@ export const constants = {
     URL_WETH_IO: 'https://weth.io/',
     URL_CANONICAL_WETH_POST: 'https://blog.0xproject.com/canonical-weth-a9aa7d0279dd',
     URL_ZEROEX_CHAT,
-    URL_LAUNCH_KIT_BACKEND: 'https://github.com/0xProject/0x-launch-kit-backend',
-    URL_LAUNCH_KIT_BLOG_POST: 'https://blog.0xproject.com/introducing-the-0x-launch-kit-4acdc3453585',
+    URL_LAUNCH_KIT: 'https://github.com/0xProject/0x-launch-kit',
+    URL_LAUNCH_KIT_ERC20_DEMO: 'https://demo.0x.org/erc20/',
+    URL_LAUNCH_KIT_ERC721_DEMO: 'https://demo.0x.org/erc721/',
```

Comment:
This one still needs to be setup by @dekz

Reply:
This one is already setup at `https://demo.0x.org/marketplace`. Needs to be updated to latest code

Reply:
Updated! Thanks